src/components/TodoItem.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo;
+  isEditing: boolean;
+  isLoading: boolean;
+};
+
+export const TodoItem:React.FC<Props> = ({
+  todo,
+  isEditing,
+  isLoading,
+}) => (
+  <div
+    className={classNames('todo', { completed: todo.completed })}
+    key={todo.id}
```

Comment:
```suggestion
  <div
    className={classNames('todo', { completed: todo.completed })}
```
There is no need to add key attribute in children component